html/objects/box.stories.js

```diff
@@ -1,3 +1,13 @@
 export default {
-  title: 'Objects|Box',
+  title: 'Components|Box',
+  decorators: [
+    story => `
+      <div class="sprk-o-Box sb-decorate">${story()}</div>
+    `,
+  ],
 };
+
+export const Default = () => (
+  `
+    <div class="sprk-o-Box"></div>
```

Comment:
Should we show all the different spacing sizes of Box? I had this similar question myself, for Stack.

Reply:
talked over chat